### DISCUSSION TITLE
Why backpropagate in linear regression?

### DISCUSSION THREAD

**Discussion by har5h1l** [mrdbourke/pytorch-deep-learning#1150] [Q&A]
I'm confused about why we backpropagate in linear regression. As per my knowledge, it's only required for models like neural networks.

**Comment by LuluW8071** [comment]
You might be referring to `Gradient Descent`. Back propagation is another way to update weights while performing linear regression via neural network. You might need to look at math for this.
